hw2/03-count.js

```diff
@@ -0,0 +1,22 @@
+const input = document.getElementById("input");
+const textContainer = document.getElementById("text-container");
+
+input.addEventListener("input", () => {
+  const keyword = input.value;
+  const text = textContainer.textContent;
+  const words = text.split(" ");
+
+  if (keyword.trim() !== "") {
+    const highlightedText = words
+      .map((word) =>
+        word.toLowerCase().includes(keyword.toLowerCase())
```

Comment:
`keyword` never changes in this scope so I think we should bring `keyword.toLowerCase()` outside of the loop to avoid re-computing that